modules/javafx.controls/src/test/java/test/javafx/scene/control/MenuItemTest.java

```diff
@@ -453,18 +454,15 @@
         assertNotNull(menuItem.acceleratorProperty());
     }
 
-    @Ignore // keyCharacter cannot be null for keyCharacterCombination
     @Test public void acceleratorCanBeBound() {
-        Modifier[] modifierArray = {};
-        KeyCombination kc = new KeyCharacterCombination(null, modifierArray);
+        KeyCombination kc = new KeyCharacterCombination("A", KeyCombination.ALT_DOWN);
         SimpleObjectProperty<KeyCombination> other = new SimpleObjectProperty<KeyCombination>(menuItem, "accelerator", kc);
         menuItem.acceleratorProperty().bind(other);
         assertEquals(kc, menuItem.getAccelerator());
     }
 
-    @Ignore
     @Test public void getUnspecifiedMnemonicParsing() {
```

Comment:
Test was incorrect. I have corrected it.
Default value of mnemonicParsing property of MenuItem is true.